ansible_collections/arista/avd/requirements.txt

```diff
@@ -0,0 +1,9 @@
+netaddr==0.7.19
+Jinja2==2.11.3
+treelib==1.5.5
+cvprac>=1.0.4
```

Comment:
cvprac should be updated to 1.0.5

Reply:
This setting will install the latest (1.0.5 as of today) if not present but will keep any existing installation if it is at least 1.0.4 and avoid breaking other scripts that could leverage a function in cvprac